tests/metagen/typegraphs/identities/py/fdk.py

```
@@ -0,0 +1,1734 @@
+# This file was @generated by metagen and is intended
+# to be generated again on subsequent metagen runs.
+
+import typing
+import io
+import re
+import uuid
+import typing
```

Ruff (0.8.2)

Redefinition of unused `typing` from line 4

Remove definition: `typing`

(F811)